### PR TITLE
Adjust padding for Interactions screen (DEV-500)

### PR DESCRIPTION
**Pull request by bojjakaseenadhreddy**
DEV-500

### CODE REVIEW

File: libs/expo/betterangels/src/lib/screens/Interactions/InteractionsHeader.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default function InteractionsHeader(props: IInteractionsHeaderProps) {
       style={{
         flexDirection: 'row',
         alignItems: 'center',
-        marginBottom: Spacings.lg,
+        marginBottom: Spacings.sm,
```
Comment (Davit-BetterAngels) [inline libs/expo/betterangels/src/lib/screens/Interactions/InteractionsHeader.tsx:22]: is this in AC?

Comment (bojjakaseenadhreddy) [reply libs/expo/betterangels/src/lib/screens/Interactions/InteractionsHeader.tsx:22]: @Davit-BetterAngels it's not in AC, but this was requested in this PR [https://github.com/BetterAngelsLA/monorepo/pull/453](https://github.com/BetterAngelsLA/monorepo/pull/453#issuecomment-2214716039)

I accidentally merged the previous PR. So, I created a new PR with the missing changes.

Comment (mikefeldberg) [reply libs/expo/betterangels/src/lib/screens/Interactions/InteractionsHeader.tsx:22]: @bojjakaseenadhreddy it's not part of this ticket but could you please update the placeholder text on line 27 from `"Search Clients"` to  `"Search interactions"`

Comment (mikefeldberg) [reply libs/expo/betterangels/src/lib/screens/Interactions/InteractionsHeader.tsx:22]: @bojjakaseenadhreddy it's not part of this ticket but could you please update the placeholder text on line 27 from `"Search Clients"` to  `"Search interactions"`

**Review by mikefeldberg — APPROVED** [review]
approved with one small out-of-scope request

**Pull request merged**